src/components/Reinforcement/reinforcement.css

```diff
@@ -0,0 +1,34 @@
+.lil-reinforcement {
+  height: 408px;
```

Comment:
too large for smaller screen...let's try to make it responsive

Reply:
I have added responsive design for small screens